ingestion-beam/src/test/java/com/mozilla/telemetry/integration/PubsubIntegrationTest.java

```diff
@@ -230,20 +230,16 @@ private void publishLines(List<String> lines) throws Exception {
     ProjectSubscriptionName subscriptionName = ProjectSubscriptionName.of(projectId,
         subscriptionId);
 
-    MessageReceiver receiver = new MessageReceiver() {
-
-      @Override
-      public void receiveMessage(PubsubMessage message, AckReplyConsumer consumer) {
-        try {
-          String encoded = Json.asString(new org.apache.beam.sdk.io.gcp.pubsub.PubsubMessage(
-              message.getData().toByteArray(), message.getAttributesMap()));
-          received.add(encoded);
-        } catch (IOException e) {
-          throw new UncheckedIOException(e);
-        }
-        consumer.ack();
+    MessageReceiver receiver = ((PubsubMessage message, AckReplyConsumer consumer) -> {
```

Comment:
Unrelated change. While updating this file, I saw IntelliJ suggesting this refactor to reduce verbosity.